realm/realm-library/src/androidTest/java/io/realm/rule/TestRealmConfigurationFactory.java

```diff
@@ -129,7 +125,16 @@ public RealmConfiguration createConfiguration(String name, Object module) {
         return createConfiguration(null, name, module, null);
     }
 
+    public RealmConfiguration createConfiguration(String name, CompactOnLaunchCallback compactOnLaunchCallback) {
```

Comment:
I agree on that point, I will remove `createConfiguration(String, String, Object, byte[], CompactOnLaunchCallback)`.